docker/daemon_test.go

```diff
@@ -34,6 +34,49 @@ func TestLoadDaemonCliConfigWithoutOverriding(t *testing.T) {
 	}
 }
 
+func TestLoadDaemonCliConfigWithDaemonFlags(t *testing.T) {
```

Comment:
@tiborvass the function  `loadDaemonCliConfig`  requires daemon flags and also config file, so this unit test includes the daemon flags, and also config file,  we can leave config file empty.  right ?